itou/www/autocomplete/views.py

```diff
@@ -125,13 +127,22 @@ def communes_autocomplete(request):
         else:
             communes = autocomplete_name(active_communes_qs, term, extra_ordering_by="code")
 
-        communes = [
-            {
-                "value": f"{commune.name} ({commune.department_code})",
-                "code": commune.code,
-                "department": commune.department_code,
-            }
-            for commune in communes[:MAX_CITIES_TO_RETURN]
-        ]
+        if select2_mode:
+            communes = [
+                {
+                    "text": commune.autocomplete_display(),
+                    "id": commune.pk,
+                }
+                for commune in communes
```

Comment:
Manque le LIMIT je crois.
```suggestion
                for commune in communes[:MAX_CITIES_TO_RETURN]
```

Reply:
corrigé